src/Common/dataAccess/readOffers.ts

```diff
@@ -6,11 +6,7 @@ import { client } from "../CosmosClient";
 import { sendCachedDataMessage } from "../MessageHandler";
 import { userContext } from "../../UserContext";
 
-export const readOffers = async (isServerless?: boolean): Promise<Offer[]> => {
-  if (isServerless) {
-    return []; // Reading offers is not supported for serverless accounts
```

Comment:
Is reading offers now supported for serverless accounts?

Reply:
No I'm just moving the check to the caller so it won't make the call if the account is serverless. It's cleaner that way because now we have another code path that goes through RP which also doesn't support serverless.

Reply:
Its is not, so we need to be sure this change does not break serverless in some way